conans/client/installer.py

```diff
@@ -312,6 +318,24 @@ def _node_concurrently_installed(self, node, package_folder):
             if node.update_manifest == read_manifest:
                 return True
 
+    def _handle_node_editable(self, node):
+        # TODO: I need a valid 'package_folder', but I'm not going to use it
+        package_folder = self._client_cache.conan(node.conan_ref)
+        self._call_package_info(node.conanfile, package_folder=package_folder)
+
+        package_layout = self._client_cache.package_layout(node.conan_ref)
+        package_layout_file = package_layout.editable_package_layout_file()
+        content = load(package_layout_file)
+        base_path = os.path.dirname(package_layout_file)
+        data = parse_editable_cpp_info(content=content, base_path=base_path,
+                                       settings=node.conanfile.settings,
+                                       options=node.conanfile.options)
+
+        # Replace directories with those in 'data'
+        cpp_info = node.conanfile.cpp_info
+        for key, items in data.items():
+            setattr(cpp_info, key, items)
```

Comment:
Maybe these three lines could be nicely moved also to an `apply_to_cpp_info` if we had a model for the `.conan_package_layout`